examples/ntrip/ntrip_example.cc

```diff
@@ -70,11 +71,9 @@ int main(int argc, char* argv[]) {
   try {
     // Check command line arguments.
     if (argc != 4) {
-      std::cerr << "Usage: http_server <address> <port> <doc_root>\n";
-      std::cerr << "  For IPv4, try:\n";
-      std::cerr << "    receiver 0.0.0.0 80 .\n";
-      std::cerr << "  For IPv6, try:\n";
-      std::cerr << "    receiver 0::0 80 .\n";
+      LOG(WARNING) << "Usage: http_server <address> <port> <doc_root>\n";
```

Comment:
Leave these at `INFO` level and add:

```
  FLAGS_logtostderror = true;
  FLAGS_colorlogtostderror = true;
```

so the program prints info stuff by default. It's so stupid that the default behavior is not to output anything.